v2/pkg/operators/common/dsl/dsl.go

```diff
@@ -513,6 +513,48 @@ func init() {
 				return float64(offset.Unix()), nil
 			},
 		),
+		"to_unix_time": makeDslWithOptionalArgsFunction(
+			"(input string, optionalLayout string) int64",
+			func(args ...interface{}) (interface{}, error) {
+				input := types.ToString(args[0])
+
+				nr, err := strconv.ParseFloat(input, 64)
+				if err == nil {
+					return int64(nr), nil
+				}
+
+				if len(args) == 1 {
+					layouts := []string{
```

Comment:
I think we can extract these into a global package shared variable to avoid allocations at each helper call.